app-main/public/css/style.css

```diff
@@ -472,15 +468,46 @@ span.lessons-count {
 }
 
 .create-lesson-input {
+  outline: none;
   margin-bottom: 20px;
   padding: 12px;
   border: 1px solid #ccc;
 }
 
-input.create-lesson-input {
+.create-lesson-input-invalid {
+  outline: none;
```

Comment:
This is fine, but when setting borders it can cause some layout shifting. `box-sizing: border-box;` means that padding and border are included in the width and height of an element. We could try using `outline` instead and see how that looks? I haven't tested it so it might look terrible.

Reply:
I'll test it out maybe it does look better.